.github/workflows/container.yml

```diff
@@ -39,4 +39,16 @@ jobs:
         run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 
       - name: build the ceph container arm64 image
-        run: make BASEOS_REGISTRY=quay.io BASEOS_TAG=stream8 RELEASE="demo" BASEOS_REPO=centos/arm64v8 DAEMON_BASE_TAG="daemon-base:demo-centos-8-aarch64" DEMO_TAG="demo:demo-centos-stream8-aarch64" DAEMON_TAG="daemon:demo-centos-stream8-aarch64" FLAVORS="main,centos-arm64,8" build
+        run: make BASEOS_REGISTRY=quay.io/centos BASEOS_TAG=stream8 RELEASE="demo" BASEOS_REPO=centos DAEMON_BASE_TAG="daemon-base:demo-centos-8-aarch64" DEMO_TAG="demo:demo-centos-stream8-aarch64" DAEMON_TAG="daemon:demo-centos-stream8-aarch64" FLAVORS="main,centos-arm64,8" build
```

Comment:
Updated because we no longer need `centos/arm64v8` and can just use `centos/centos`

Reply:
Also, because of the shaman issue I mention in the description, I expect this to fail on `main` at least on the first run, but it may not be a consistent failure.

Reply:
I realize this is succeeding because the github runner is still an x86 system.

It might be good to make the arm builds do so via emulation, but it may be a fair bit of work: https://wiki.debian.org/QemuUserEmulation